docs/http-response-status.rst

```diff
@@ -1,25 +1,86 @@
+* "The given custom ids cannot contain semicolons, if you want to use csv as output format."
+* "The interval (period) of the provided time parameter is not ISO-8601 conform."
+* "The provided time parameter is not ISO-8601 conform."
+* "Invalid filter syntax. Please look at the additional info and examples about the filter parameter at https://docs.ohsome.org/ohsome-api. Detailed error message: [...]"
+* "One or more boundary object(s) have a custom id (or at least a colon), whereas other(s) don't. You can either set custom ids for allyour boundary objects, or for none."
+* "Unsupported content-type header found. Please make sure to use either 'multipart/form-data' or 'application/x-www-form-urlencoded'."
+* "The filter2 parameter has to be defined when using a /ratio endpoint."
+* "The given 'format' parameter is invalid. Please choose between 'geojson'(only available for /groupBy/boundary and data extraction requests), 'json', or 'csv'."
+* "Error in processing the boundary parameter. Please remember to follow the format, where you separate every coordinate with a comma, each boundary object with a pipe-sign and add optional custom ids to every first coordinate with a colon."
+* "Unknown parameter '[parameter]' for this resource."
+* "Unknown parameter '[parameter]' for this resource. Did you mean '[parameter]'?"
+* "Unknown parameter '[parameter]' for this resource. Did you mean '[parameter]' or '[parameter]'?"
+* "Every parameter has to be unique. You can't give more than one '[parameter type]' parameter."
+* "There cannot be more input values in the values|values2 than in the keys|keys2 parameter, as values_n must fit to keys_n."
+* "Parameter 'types' (and 'types2') cannot have more than 4 entries."
+* "The given timeout does not fit to its format. Please give one value in seconds and use a point as the decimal delimiter, if needed."
+* "The given timeout is too long. It has to be shorter than [timeout] seconds"
+* "You need to give at least two timestamps or a time interval for this resource."
+* "The given parameter [parameter] can only contain the values 'true', 'yes', 'false', or 'no'."
+* "Your provided boundary parameter (bboxes, bcircles, or bpolys) does not fit its format, or you defined more than one boundary parameter."
+* "The geometry of your given boundary input could not be parsed for the creation of the response GeoJSON."
+* "The keys, values and types parameters must be empty, when you set the filter parameter."
+* "The properties parameter of this resource can only contain the values 'tags' and/or 'metadata' and/or 'contributionTypes' and/or 'unclipped'."
+* "The properties parameter of this resource can only contain the values 'tags' and/or 'metadata' and/or 'unclipped'."
+* "Parameter 'types' (and 'types2') can only have 'node' and/or 'way' and/or 'relation' OR 'point' and/or 'line' and/or 'polygon' and/or 'other'"
```

Comment:
I don't know how useful a list of possible messages is, to be honest. Since they are just listed here plainly without further explanation, there's not really much users could learn from this list, could they? If anything, the list would be quite cumbersome to maintain (i.e. keep up-to-date) for us.

Instead one could simply write that: _“The error message contains further details about why the query failed.”

Or what was your reasoning behind including it? Maybe I'm simply not seeing it at the moment.

Reply:
I understood #150 in that way. @joker234 Am I wrong? If yes, no problem, I can change it summarizing the reasons for every  HTTP response code of the ohsome API.

Reply:
hm, ok. I would say it might be better to mention the possible cases in a slightly abstracted way, instead of a plain verbatim list. e.g. something like this:

```
* ``400 Bad Request`` - the ohsome API cannot or will not process the request due to a client error. This can occur because a parameter is not given in the correct format,  mandatory parameters are not present or other malformed requests. The error message contains further details about why the query failed.
* ``404 Not Found``-  the requested resource could not be found. This can occur when an invalid resource is requested, or when the provided boundary or time parameter does is not covered by the underlying data OSHDB database.
* ``413 Payload Too Large`` - the request is larger than the ohsome API is willing or able to process. Occurs when the given query took takes too long to compute.
```

Reply:
I'm not quite sure what the intention should be. I would agree it doesn't make sense to joust copy & paste all error messages as you have to keep track of them, when you change the ohsome API's code. But maybe a little bit better explanation in the ohsome API documentation would be nice. Alternatively one could think about having more actual error codes in one list as the ORS has. https://github.com/GIScience/openrouteservice/blob/master/openrouteservice/docs/services/error_codes.md

Reply:
I have updated this part trying to improve the explanation without the actual messages.